third_party/ccid/patches/0002-Create-ACS_ACR1252IMP_Reader.txt.patch

```diff
@@ -0,0 +1,124 @@
+From 6c0abdec5025d00495c2a4835496c54f5fe0e439 Mon Sep 17 00:00:00 2001
+From: ankhler <47143834+ankhler@users.noreply.github.com>
+Date: Thu, 24 Sep 2020 08:59:01 +0200
+Subject: [PATCH] Create ACS_ACR1252IMP_Reader.txt
+
+---
+ .../src/readers/ACS_ACR1252IMP_Reader.txt     | 105 ++++++++++++++++++
+ 1 file changed, 105 insertions(+)
+ create mode 100644 third_party/ccid/src/readers/ACS_ACR1252IMP_Reader.txt
```

Comment:
Ideally, the "supported_readers.txt" file should be in the patch file as well. It's present in the upstream commit as well: https://github.com/LudovicRousseau/CCID/commit/183c09cdc0973ba63fefe7e01a3b4e9982dcf3ab#diff-cea437f229b9b64905a018a5bd92fdde